RoomsEyeView/Logic/vtkSlicerRoomsEyeViewModuleLogic.cxx

```diff
@@ -700,12 +700,6 @@ vtkSlicerRoomsEyeViewModuleLogic::SetupTreatmentMachineModels(vtkMRMLRoomsEyeVie
     vtkErrorMacro("SetupTreatmentMachineModels: Invalid scene");
     return std::vector<TreatmentMachinePartType>();
   }
-  vtkSlicerBeamsModuleLogic* beamsLogic = this->GetBeamsLogic();

```

Comment:
If this was not necessary, can you please review the rest of these in this file? I saw that you added about half a dozen of these. Just to keep the file as readable as possible... Thank you!

Reply:
Mmm I picked all of them from the IEC branch from Mark, but I believe they were removed in the meanwhile in master, so I guess I can safely remove them here, too.

Reply:
Right, I guess I removed these recently indeed

Reply:
Removed.